keps/sig-node/4815-dra-partitionable-devices/README.md

```diff
@@ -647,7 +742,124 @@ When such a device is allocated, the scheduler will need to track the full
 capacity required to satisfy each of the sink devices along the chain. In this
 way, all intermediate sink devices will essentially be rendered
 "unschedulable", with the last-level sink device pulling its capacity from the
-devices it references directly.
+device it references directly.
+
+### Defining multi-host devices
+
+An example of a small 4x4 TPU slice with its partitions will look like the
+following:
+
+```yaml
```

Comment:
Who publishes this? A centralized controller I assume?

Reply:
Yeah, that is the idea. I think that is probably the case for any `ResourceSlice` that has a different node selection than `NodeName`. I've also added sentence to the example to clarify this.